components/automate-chef-io/content/docs/ldap.md

```diff
@@ -36,8 +36,7 @@ require you to adjust the [IAM]({{< relref "iam-v2-overview.md" >}}) policy memb
 
 {{< info >}}
 Users who sign in via SAML will have a session time of 24 hours before needing to sign in again.
```

Comment:
idk why we have this in the ldap docs, can we remove it?

Reply:
I'm pretty sure this warning is a hold-over from when SAML and LDAP were together in the same configuration document.